ScoutSuite/providers/azure/rules/findings/sqldatabase-databases-no-auditing.json

```diff
@@ -1,23 +1,36 @@
 {
-    "description": "Auditing Disabled for SQL Databases",
-    "rationale": "Enable auditing for all SQL databases.",
-    "compliance": [
-        {
-            "name": "CIS Microsoft Azure Foundations",
-            "version": "1.0.0",
-            "reference": "4.2"
-        }
-    ],
-    "dashboard_name": "SQL Databases",
-    "display_path": "sqldatabase.subscriptions.id.servers.id",
-    "path": "sqldatabase.subscriptions.id.servers.id.databases.id",
-    "conditions": [
-        "and",
-        [
-            "sqldatabase.subscriptions.id.servers.id.databases.id.auditing.auditing_enabled",
-            "false",
-            ""
-        ]
-    ],
-    "id_suffix": "db_auditing_disabled"
+  "description": "Auditing Disabled for SQL Databases",
+  "rationale": "Enable auditing for all SQL databases.",
```

Comment:
Can you add a bit of wording around what Auditing for SQL dbs is?

Reply:
Yes, i've added a rationale for auditing as well as for data encryption and low retention.